src/scala/scripts/TwitterScroogeGenerator.scala

```diff
@@ -41,7 +41,7 @@ class ScroogeGenerator extends Processor {
 
   def processRequest(args: java.util.List[String]) {
     def getIdx(i: Int): List[String] =
-      if (args.size > i) args.get(i).split(':').toList.filter(_.nonEmpty)
+      if (args.size > i) args.get(i).drop(1).split(':').toList.filter(_.nonEmpty)
```

Comment:
Can we extract 1 into a constant as well to express why we're dropping it? Something like .drop(PaddingCharacterUsedForArgumentPreservation)
I know the names are long but honestly I fear this will cryptic code really fast